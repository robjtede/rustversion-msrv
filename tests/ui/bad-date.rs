#[rustversion_msrv::nightly(stable)]
struct S;

#[rustversion_msrv::any(nightly(stable))]
struct S;

fn main() {}
