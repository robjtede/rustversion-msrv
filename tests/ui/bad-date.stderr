error: expected nightly date, like 2024-05-12
 --> tests/ui/bad-date.rs:1:29
  |
1 | #[rustversion_msrv::nightly(stable)]
  |                             ^^^^^^

error: expected nightly date, like 2024-05-12
 --> tests/ui/bad-date.rs:4:33
  |
4 | #[rustversion_msrv::any(nightly(stable))]
  |                                 ^^^^^^
